orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java

```diff
@@ -92,14 +155,16 @@ public UserConfiguredUrlRestrictions build() {
   private final boolean rejectLinkLocal;
   private final Set<String> rejectedIps;
   private final HttpClientProperties clientProperties;
+  private final List<Pattern> excludedPatterns;
 
-  public UserConfiguredUrlRestrictions(
+  protected UserConfiguredUrlRestrictions(
```

Comment:
The visibility of this constructor was reduced to mandate the use of the `Builder`